src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -13,8 +13,24 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <!-- Documentation warnings -->
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <TargetsForTfmSpecificBuildOutput>GatherProjectReferenceAssets</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
-  
+
+  <!-- The following properties are set to package M.D.A.Interactive with the M.D.A nuget package. If M.D.A.I undergoes TFM or dependency changes, we need to update the dependencies below-->
+  <Target Name="GatherProjectReferenceAssets" Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <MSBuild Projects="./../Microsoft.Data.Analysis.Interactive/Microsoft.Data.Analysis.Interactive.csproj"
+             Targets="_GetBuildOutputFilesWithTfm"
+             Properties="TargetFramework=$(TargetFramework)">
+      <Output TaskParameter="TargetOutputs" ItemName="BuildOutputInPackage" />
+    </MSBuild>
+  </Target>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <PackageReference Include="HtmlAgilityPack" Version="1.11.24" />
+    <PackageReference Include="Microsoft.DotNet.Interactive" Version="1.0.0-beta.20154.7" />
```

Comment:
This isn't correct. The `Microsoft.Data.Analysis` package shouldn't have a dependency on `Microsoft.DotNet.Interactive`, on any TFM.

Reply:
We added it because this package would now also ship Microsoft.Data.Analysis.Interactive on the lib so we had to also bring in it's dependencies. From what I'm reading in your comments that is not what we wanted apparently and we wanted M.D.A.I to instead be placed in a different folder, in that case should it be packaged as a tool? (in other words, should it carry its dependencies like MDI with it so that it doesn't have a package dependency on that?)

Reply:
> From what I'm reading in your comments that is not what we wanted apparently and we wanted M.D.A.I to instead be placed in a different folder, in that case should it be packaged as a tool? (in other words, should it carry its dependencies like MDI with it so that it doesn't have a package dependency on that?)

The M.D.A.I. assembly will only be loaded by `dotnet-interactive`, so we don't need to carry those dependencies in our package. Those dependencies will be provided by `dotnet-interactive` itself.

@jonsequitur @colombod - do we have any documentation explaining the dotnet-interaction extension nupkg layout and expectations? I see https://github.com/dotnet/interactive/blob/master/docs/extending-dotnet-interactive.md, but that doesn't explain packaging expectations.